### PR TITLE
3222 Update Coverage

### PR DESCRIPTION
**Pull request by flooie**
PR includes more updates 

Spinner for long running tasks
Adds 7 Day cache from 1 day

Updates language on coverage pages and table of contents
Cleans up various sentences and style choices

Adds support for mobile and dynamically adjusts margins 
as well as resize the chart for a broad range of sizes

Drops the sidebar on mobile - and switches (to court IDs) on small
screens.  

We do not have consistent small enough labels so ID is used.  

Removed chartographer.js and coverage.js in simple pages

### CODE REVIEW

**Review by mlissner — CHANGES_REQUESTED** [review]
Thanks Bill. I haven't looked too deeply at the JS, I'm hoping @ERosendo could do that, but I do have a couple other comments. 

Also, you can remove plottable and d3 from the page too, right?

Finally, can you share some screenshots?

What is id=json-data?

File: cl/simple_pages/templates/help/coverage_opinions.html [hunk 1]
```diff
@@ -60,31 +60,41 @@ <h3>Table of Contents</h3>
...
+        opacity:0;
+        transition: opacity 200ms ease-in;
+    }
+    .htmx-request .htmx-indicator{
+        opacity:1
+    }
+    .htmx-request.htmx-indicator{
+        opacity:1
+    }
+    </style>
```
[the remark this answers is not in the document]
Comment (flooie) [reply cl/simple_pages/templates/help/coverage_opinions.html:90]: that wasn't supposed to be there.

**Comment by flooie** [comment]
This is the top when the dimensions are smaller than an iPhone.  

Screenshot 2023-10-11 at 6 16 53 PM

The dreaded Ohio Examples 
Each of which is exaggerated by my ability to take super long screenshots but I felt it useful.  
Screenshot 2023-10-11 at 6 19 57 PM
Screenshot 2023-10-11 at 6 20 11 PM
Screenshot 2023-10-11 at 6 20 19 PM


Finally some standard shots one with names and one with IDs 
Screenshot 2023-10-11 at 6 19 10 PM
Screenshot 2023-10-11 at 6 19 30 PM

**Review by ERosendo — APPROVED** [review]
The JS code looks good! 👍

File: cl/simple_pages/templates/help/coverage.html [hunk 2]
```diff
@@ -10,10 +10,8 @@
 
     {% if DEBUG %}
         <script src="{% static "js/d3.js" %}"></script>
```
Comment (mlissner) [inline cl/simple_pages/templates/help/coverage.html:12]: Are we still using d3 on this page?

**Review by mlissner — APPROVED** [review]
Cool, let's give it a go.

File: cl/simple_pages/templates/help/coverage.html (hunk 2 above)
Comment (flooie) [reply cl/simple_pages/templates/help/coverage.html:12]: sure doesnt look like it

**Pull request merged**